package.json

```diff
@@ -171,7 +171,6 @@
     "nerdamer": "^1.1.4",
     "ng-infinite-scroll": "^1.3.0",
     "ngx-cookie": "^5.0.2",
-    "ngx-infinite-scroll": "^10.0.1",
```

Comment:
Why is this removed?

Reply:
Not sure how this got removed, I've undone the changes!